src/sign_workflow/sign_args.py

```diff
@@ -14,12 +14,14 @@
 class SignArgs:
     ACCEPTED_SIGNATURE_FILE_TYPES = [".sig", ".asc"]
     ACCEPTED_PLATFORM = ["linux", "windows"]
+    ACCEPTED_OVERWRITE_VALUES = ["True", "False"]
```

Comment:
This should be a plain Boolean arg without a value and used with just `—overwrite`

Reply:
done!